examples/cpp/infer_multiple_networks/inference_engine.cc

```diff
@@ -40,28 +40,30 @@ extern tf::Status LoadGraph(const string& graph_file_name,
                             std::unique_ptr<tf::Session>* session,
                             const tf::SessionOptions& options);
 
-extern tf::Status ReadTensorFromImageFile(const string& file_name,
+extern tf::Status ReadTensorFromImageFile(const std::vector<string>& file_names,
                                           const int input_height,
                                           const int input_width,
                                           const float input_mean,
                                           const float input_std, bool use_NCHW,
+                                          const int input_channels,
                                           std::vector<tf::Tensor>* out_tensors);
 
 namespace infer_multiple_networks {
 
 InferenceEngine::InferenceEngine(const string& name, const string& backend)
     : m_name(name) {}
-Status InferenceEngine::Load(const string& network, const string& image_file,
-                             int input_width, int input_height,
-                             float input_mean, float input_std,
+Status InferenceEngine::Load(const string& network,
+                             const std::vector<string>& image_files,
+                             const int& input_width, const int& input_height,
+                             const float& input_mean, const float& input_std,
                              const string& input_layer,
                              const string& output_layer, bool use_NCHW,
-                             bool preload_images) {
+                             bool preload_images, const int& input_channels) {
```

Comment:
Why do you have const int&? just an `int` would do - no?

Reply:
It would. Few were const T& and few weren't here so made all of them same, will modify.